pattern/Disjunction.java

```diff
@@ -96,4 +102,8 @@ public boolean equals(Object o) {
     public int hashCode() {
         return hash;
     }
+
+    public Set<Identifier.Variable.Retrievable> allBranchesRetrieve() {
+        return allBranchesRetrieve;
+    }
```

Comment:
Convenience function (maybe this isn't the right place) to determine the variables which are common to all branches.